src/scratch/kingkong/kingkong.md

```diff
@@ -61,9 +64,9 @@ skyskraper, et fly og en gorilla.
   med de andre bildene, slik at du har tre figurer, en skyskraper, et
   fly og en King Kong-figur.
 
-+ Figurene får navn fra bildefilene. Endre gjerne navnet på figurene
-  under `Drakter` til `skyskraper`, `fly` og `kong` slik at det blir enklere å
-  holde oversikten.
++ Figurene får navn fra bildefilene. Endre gjerne navnet på figurene i
+  figurlisten til `skyskraper`, `fly` og `kong` slik at det blir
```

Comment:
Ja.

Men dette har de gjort mange ganger allerede, så siden dette er en oppgave for erfarne har jeg ikke vært så eksplisitt på det.